lk2nd/device/dts/msm8952/msm8920-motorola-jeter.dts

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <skeleton64.dtsi>
+#include <lk2nd.dtsi>
+
+/ {
+	qcom,msm-id = <QCOM_ID_MSM8920 0x00>;
+	qcom,board-id = <0x43 0x8000>,
+			<0x43 0x83a0>;
+
+	/* 
+	 * model is required by bootloader to pick dtb.
+	 * The bootloader also crashes if model isn't present in every dtb. (see lk2nd.dtsi for more info)
+	 * Build with: make TOOLCHAIN_PREFIX=arm-none-eabi- SIGN_BOOTIMG=1 LK2ND_ADTBS="msm8920-motorola-jeter.dtb" LK2ND_DTBS="" lk2nd-msm8952
+	 */
+	
+	model = "jeter";
+};
+
+&lk2nd {
+	model = "Motorola Moto G6 Play (jeter) (MSM8920)";
+	compatible = "motorola,jeter";
+	lk2nd,match-device = "jeter";
+
+	lk2nd,dtb-files = "msm8937-motorola-jeter";
+
+	gpio-keys {
+		compatible = "gpio-keys";
+
+		up {
+			lk2nd,code = <KEY_VOLUMEUP>;
+			gpios = <&tlmm 91 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>;
+		};
+
+		down {
+			lk2nd,code = <KEY_VOLUMEDOWN>;
+			gpios = <&pmic_pon GPIO_PMIC_RESIN 1>;
+		};
+
+		power {
+			lk2nd,code = <KEY_POWER>;
+			gpios = <&pmic_pon GPIO_PMIC_RESIN 0>;
+		};
```

Comment:
The third value for pmic_pon is non-sensical as it represents flags, which don't matter for pmic_pon. You set both keycodes to the same resin key (which is, usually, vol-) Perhaps just keeping override for vol+ is enough? those other two seem to be default anyway.